### ISSUE TITLE
Working with {{ }} shortcodes to output raw elements, without <div class=""> tags

### ISSUE THREAD

**Issue by DerekMoss95** [slimndap/wp-theatre#256]
Hello Jeroen! 

I am having an issue with the short-codes when trying to produce a lightbox for youtube videos on my school's site. Here is the code I have written for the data we want displayed: 

```
[wpt_productions start="2017-01-01" end="2017-04-20"]{{thumbnail}}

<h3>{{title|permalink}}</h3>

{{language}} / {{year}} / {{duration}} min.</br>

Director: {{director}} </br>

<a class=reverse href={{trailer}} rel=wp-video-lightbox>Trailer</a>

{{content}}

[/wpt_productions]
```

The {{trailer}} shortcode needs to output just the raw link that we have inserted into the acf field, but it outputs `<div class="wp_theatre_prod_trailer">https://www.youtube.com/watch?v=Go63ZVVwivI</div>` for example. 

I can obviously get rid of the `<div class="wp_theatre_prod_trailer"> and </div>` tags by getting rid of it in the function custom( $field, $args = array() ) {} function on line 812 of wpt_production.php in the second if statement, but then I cannot update the Cinema plugin you have built because it will overwrite my changes to the plugin. 

Is there a way of outputting the raw element using that shortcode/lightbox code without editing anything in the plugin? 

Is there any other way? Possibly in the theme code? 

Thank you in advance. 

Sincerely, 
Derek Moss

**Comment by slimndap** [comment]
You can overwrite the HTML output of custom fields by using the `wpt_production_$field_html` filter. So in your case it would be the `wpt_production_trailer_html` filter.

You can find it here:
https://github.com/slimndap/wp-theatre/blob/master/functions/wpt_production.php#L836

**Comment by DerekMoss95** [comment]
Thank you so much Jeroen! 

I had to teach myself about WordPress filters, hooks, actions, and the apply_filters() and add_filters() functions all day to figure it out but I was able to figure it out using your tip! I was able to strip the tags out of the HTML output with the `wp_strip_all_tags()` function in my theme's functions.php file and it works great! I will post my function here for other's future reference:

In my theme's function.php file I wrote the following function:

```
function strip_tags_filter($html) {
	$html = wp_strip_all_tags($html, $remove_breaks=true);
	return $html;
}
add_filter('wpt_production_trailer_html', 'strip_tags_filter', 10, 2);
```

Thanks again Jeroen, you saved me loads of time and effort with your tip.  Take care!

Best regards,
Derek Moss
